src/main/java/app/backend/controller/optimization/OptimizationController.java

```diff
@@ -74,4 +72,18 @@ public ResponseEntity<OptimizationResultResponse> getOptimizationResult(
     ) {
         return optimizationUtils.retrieveOptimizationResult(crossroadId, day, hour);
     }
+
+    @PostMapping(value = "/base")
+    public ResponseEntity<Void> addTrafficLightsCycles(
+            @RequestParam("file") MultipartFile trafficLightsCycles,
+            @RequestParam("crossroadId") String crossroadId,
```

Comment:
Możemy to zrobić path param? Zwykle tak jest z crossroadId i też bym chyba tak wolał